File: 2574-left-and-right-sum-differences/2574-left-and-right-sum-differences.java
class Solution {
    public int[] leftRightDifference(int[] nums) {
       int n=nums.length;
       int l[]=new int[n];
       int r[]=new int[n];
       l[0]=0;
       r[n-1]=0;
       for(int i=1;i<n;i++){
        l[i]=l[i-1]+nums[i-1];
       } 
       for(int i=n-2;i>=0;i--){
        r[i]=r[i+1]+nums[i+1];
       }
       int f[]=new int[n];
       for(int i=0;i<n;i++){
        f[i]=Math.abs(l[i]-r[i]);
       }
       return f;
    }
}

File: 2574-left-and-right-sum-differences/2574-left-and-right-sum-differences.cpp
class Solution {
public:
    vector<int> leftRightDifference(vector<int>& nums) {
       int n=nums.size();
       int s=0;
       for(int i=0;i<n;i++){
        s+=nums[i];
       }
       int l=0;
       vector<int> res;
       for(int i=0;i<n;i++){
        s-=nums[i];
        res.push_back(abs(l-s));
        //s-=l;
        l+=nums[i];
       }
       return res;
    }
};